hphp/runtime/vm/jit/vasm-simplify.cpp

```diff
@@ -438,6 +438,7 @@ void simplify(Vunit& unit) {
 
     for (auto const& inst : blocks[b].code) {
       visitDefs(unit, inst, [&] (Vreg r) { env.def_insts[r] = inst.op; });
+      // Following line segfaults
```

Comment:
After I doctored up the check SSA code, this started to fail. I threw my hands up and just switched the code in vasm-lower.cpp to pass the odd stack argument twice.